modular_splurt/code/game/objects/structures/bed_chairs/sofa.dm

```diff
@@ -0,0 +1,5 @@
+/obj/structure/chair/sofa/corp
+	icon = 'modular_splurt/icons/obj/sofa.dmi'
+
+/obj/structure/chair/sofa/corp/corner/handle_layer() //only the armrest/back of this chair should cover the mob.
+	return
```

Comment:
Small issue
![dreamseeker_GoQZAeW92k](https://user-images.githubusercontent.com/93835010/198813032-f361453a-5bf3-4a2c-911f-f7633acf57a0.gif)

Reply:
hgnngghhstt'veee
I literally tested this but I guess my VSC must've not finished compiling or something ;_;